### PR TITLE
Added to participant

### PR DESCRIPTION
**Pull request by oatrice**
#1 referenced!

### CODE REVIEW

File: paticipants/oatrice.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Name : Anirut Teerabut
```
Comment (MJ10) [inline paticipants/oatrice.txt]: Delete this file. It is a duplicate.

File: Participant/oatrice.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Name : Anirut Teerabut
```
Comment (MJ10) [inline Participant/oatrice.txt]: Delete this file too

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

**Comment by salman-bhai** [comment]
spam

**Pull request closed without merging**